crates/bevy_input/src/lib.rs

```diff
@@ -84,12 +88,39 @@ impl Plugin for InputPlugin {
                 CoreStage::PreUpdate,
                 touch_screen_input_system.label(InputSystem),
             );
+
+        // Register reflected types
+        app.register_type::<ButtonState>();
+        app.register_type::<KeyboardInput>()
+            .register_type::<KeyCode>()
+            .register_type::<ScanCode>();
+        app.register_type::<MouseButtonInput>()
+            .register_type::<MouseButton>()
+            .register_type::<MouseMotion>()
+            .register_type::<MouseScrollUnit>()
+            .register_type::<MouseWheel>();
+        app.register_type::<TouchInput>()
+            .register_type::<ForceTouch>()
+            .register_type::<TouchPhase>();
+        app.register_type::<Gamepad>()
```

Comment:
Super Minor Nit: I see why you did this. The grouping might be a bit clearer either with spaces between each group and/or a comment above each group.

For example:
```suggestion
        // Button types
        app.register_type::<ButtonState>();
        // Keyboard types
        app.register_type::<KeyboardInput>()
            .register_type::<KeyCode>()
            .register_type::<ScanCode>();
        // Mouse types
        app.register_type::<MouseButtonInput>()
            .register_type::<MouseButton>()
            .register_type::<MouseMotion>()
            .register_type::<MouseScrollUnit>()
            .register_type::<MouseWheel>();
        // Touch types
        app.register_type::<TouchInput>()
            .register_type::<ForceTouch>()
            .register_type::<TouchPhase>();
        // Gamepad types
        app.register_type::<Gamepad>()
```